File: pe/nonvisualstudio/clojure/Implementation/Problem0007.clj
(ns pe.clojure.implementation.problem0007)

(load-file "C:/Users/Thomas/Documents/GitHub/practice/utilities/nonvisualstudio/clojure/Implementation/Primes.clj")
(use '[utilities.primes :as util.primes])

(defn nthPrime [nthPrime]
  (nth
    (util.primes/lazyPrimes 2)
    (- nthPrime 1)))

File: pe/nonvisualstudio/elixir/Implementation/Problem0007.ex
Code.require_file "../../../../Utilities/nonvisualstudio/elixir/Implementation/Primes.ex", __DIR__

defmodule Problem0007 do
  def nthPrime(nth) do
    Primes.generatePrimes(nth * 11) |> Enum.at(nth - 1)
  end
end
